libexec/revng/revng-check-conventions

```diff
@@ -1,498 +1,727 @@
+            sys.exit(1)
+    return result
+
+
+def get_file_list(args) -> List[Path]:
+    if len(args.FILES) > 0:
+        files = args.FILES
+        for file in files:
+            if not os.path.isfile(file):
+                print(f"File {file} does not exist", file=sys.stderr)
+                sys.exit(1)
+    elif args.HEAD:
+        files = git_diff_wrapper("HEAD^..HEAD")
+    elif args.commit_range:
+        files = git_diff_wrapper(args.commit_range)
+    elif args.cached:
+        files = git_diff_wrapper("--cached")
+    elif args.all:
+        files = git_ls_files_wrapper()
+    else:
+        files = git_ls_files_wrapper("-m", "--exclude-standard")
+    return [Path(f) for f in files]
+
+
+def read_file(filename, size: Optional[int] = None):
+    with open(filename) as f:
+        content = f.read(size)
+        return (filename, content)
+
+
+def read_header(filename):
+    return read_file(filename, 512)
+
+
+def read_clang_style(args) -> str:
+    if not args.use_local_clang_format_file:
+        style = get_config("clang-format-style-file").read_text()
+    else:
+        style = "file"
+    if args.print_clang_format_config:
+        process_run(["clang-format", "--dry-run", f"-style={style}", "--dump-config"])
+        sys.exit(0)
+    return style
+
+
+def inflate_files(initial_dict: Dict[str, Dict[Path, str]]) -> Dict[str, Dict[Path, str]]:
+    paths: Set[Path] = set()
+    for pathset in initial_dict.values():
+        paths.update(pathset.keys())
+    contents = {path: content for path, content in run_parallel(read_file, paths)}
+    return {
+        tag: {path: contents[path] for path in paths.keys()} for tag, paths in initial_dict.items()
+    }
+
+
+WRITE_PASSES: Tuple[Pass, ...] = (
+    ThreadedCommandPass(
+        "clang-format",
+        "c",
+        ("clang-format", "-Werror", "-style=${clang_style}", "-i"),
+        ("clang-format", "-Werror", "--dry-run", "-style=${clang_style}", "-i"),
+    ),
+    ThreadedCommandPass(
+        "cmake-format",
+        "cmake",
+        ("cmake-format", "-l", "error", "-i"),
+        ("cmake-format", "-l", "error", "--check"),
+    ),
+    SingleCommandPass(
+        "black", "python", ("black", "-q", "-l", "100"), ("black", "-q", "--check", "-l", "100")
+    ),
+    SingleCommandPass(
+        "isort",
+        "python",
+        ("isort", "--settings-file", "$isort_config"),
+        ("isort", "--settings-file", "$isort_config", "--diff"),
+    ),
+    SingleCommandPass(
+        "prettier",
+        "prettier",
+        ("prettier", "--write", "--config", "$prettier_config", "--loglevel", "silent"),
+        ("prettier", "--check", "--config", "$prettier_config", "--loglevel", "silent"),
+    ),
+)
+
+READ_PASSES: Tuple[Pass, ...] = (
+    SingleCommandPass("flake8", "python", ("flake8", "--config", "$flake8_config")),
+    MyPyPass(),
+    LicenseCheckPass(),
+    BashPass(),
+    RevngCPass(),
+    RevngGeneralPass(),
```

Comment:
Can you drop `Revng` prefix and get more descriptive names?

Reply:
Done